migrations/1_check_authority_address.js

```diff
@@ -1,10 +1,36 @@
 require('dotenv').config()
-var RootChain = artifacts.require('./RootChain.sol')
+const RootChain = artifacts.require('./RootChain.sol')
+
 
 module.exports = async function (deployer, network, accounts) {
-  if (process.env.USE_EXISTING_AUTHORITY_ADDRESS) {
+  if (process.env.AUTHORITY_PRIVATEKEY && process.env.DEPLOYER_PRIVATEKEY) {
```

Comment:
Hmmm, there seems to be quite a bit of code duplication in this block: checking auth nonce and funding the auth account. 
Can we rearrange the logic to avoid duplication?

Reply:
Done! https://github.com/omisego/plasma-contracts/pull/220/commits/65d0f9757201122250a4c1e84d6b7c49bc15d0e3